serverless.yml

```diff
@@ -0,0 +1,19 @@
+service: nookwill
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: us-west-2
+
+package:
+ include:
+   - backend/handler.js
+ exclude:
+   - tf/**
+   - .gitignore
+   - my-app/**
```

Comment:
this folder doesn't exist so we don't need this line. It did make me realize our folder structure is a little off though so kinda glad you had this in here haha